### ISSUE TITLE
Provide easy way to enable logging.

### ISSUE THREAD

**Issue by grokys** [AvaloniaUI/Avalonia#845]
Logging to trace output should be enabled on debug builds by default in the application template. At the moment it's a bit tricky to set up. We need to be able to do something like:

```
AppBuilder.Configure<App>()
    .UsePlatformDetect()
    .EnableLogging(AvaloniaLogEventLevel.Verbose)
    .Start<MainWindow>();
```

This will probably require moving to .NET standard as the trace output is not available on our current PCL profile.

**Comment by kekekeks** [comment]
We also need a way to format log strings without referencing Serilog. We can't just use string.Format, unfortunately

**Comment by jkoritzinsky** [comment]
Fixed by #1273
